src/main/java/com/bookstore/controller/impl/BookControllerImpl.java

```diff
@@ -0,0 +1,40 @@
+package com.bookstore.controller.impl;
+
+import com.bookstore.controller.BookController;
+import com.bookstore.dto.BookDto;
+import com.bookstore.dto.CreateBookRequestDto;
+import com.bookstore.service.BookService;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(value = "/api/books")
```

Comment:
consider extracting this common URL prefix to properties: `server.servlet.context-path=/api`